workflows/general/alignment-post.wdl

```diff
@@ -0,0 +1,61 @@
+version 1.0
+
+import "../../tools/deeptools.wdl"
+import "../../tools/samtools.wdl"
+import "../../tools/picard.wdl"
+import "../../tools/md5sum.wdl"
+import "https://raw.githubusercontent.com/stjude/XenoCP/3.1.4/wdl/workflows/xenocp.wdl" as xenocp_workflow
+
+workflow alignment_post {
+    input {
+        File bam
+        File? contaminant_db
+        String xenocp_aligner = ""
+        Boolean cleanse_xenograft = false
```

Comment:
We should add `parameter_meta` for these three. If I were an external user, I would probably be confused.